src/leappto/cli.py

```diff
@@ -494,8 +527,8 @@ def _rsync():
                                  .format(rsync_dir, ' '.join(self.target_cfg), self.target_addr)
                     Popen(shlex.split(target_cmd)).wait()
 
-                # temporary, after task with different names for containers should be removed
-                shutil.rmtree(rsync_dir)
+                    # temporary, after task with different names for containers should be removed
+                    shutil.rmtree(rsync_dir)
```

Comment:
Is it ok? I mean now source fs dir will be removed only when target is different than localhost. I wanted to avoid situation where you have mixed files from more than one machine. I know common files are being replaced, but you can have some extra files from different source, for example mysql conf if it's running on specific machine etc.